models/attention_gan_model.py

```diff
@@ -13,6 +13,8 @@ def modify_commandline_options(parser, is_train=True):
             parser.add_argument('--lambda_A', type=float, default=10.0, help='weight for cycle loss (A -> B -> A)')
             parser.add_argument('--lambda_B', type=float, default=10.0, help='weight for cycle loss (B -> A -> B)')
             parser.add_argument('--lambda_identity', type=float, default=0.5, help='use identity mapping. Setting lambda_identity other than 0 has an effect of scaling the weight of the identity mapping loss. For example, if the weight of the identity loss should be 10 times smaller than the weight of the reconstruction loss, please set lambda_identity = 0.1')
+            parser.add_argument('--nb_attn', type=int, default=10, help='nombre de masques d\'attention')
```

Comment:
let's put the help description in english

Reply:
Done !